asset-transfer-sbe/README.md

```diff
@@ -1,19 +1,51 @@
 # State-based endorsement asset transfer sample
 
-Transactions that are submitted to Hyperledger Fabric networks need to be endorsed by peers that are joined to a channel before the transaction can be added to the ledger. Fabric peers endorse transactions by executing a smart contract using the inputs of the transaction proposal. The peers then sign the input and output generated by the smart contract execution. The endorsement policy specifies the set of organizations whose peers need to endorse a transaction before it can be added to the ledger.
-
-Each chaincode that is deployed to a channel has an endorsement policy that governs the assets managed by the chaincode smart contracts. However, you can override the chaincode level endorsement policy to create an endorsement policy for a specific key, either on the public channel ledger or in a private collection. State-based endorsement policies, also known as key-level endorsement policies, allow channel members use different endorsement policies for assets that are managed by the same smart contract. For more information about endorsement policies and state-based endorsement, visit the [Endorsement Policies](https://hyperledger-fabric.readthedocs.io/en/master/endorsement-policies.html) topic in the Fabric documentation.
+Transactions that are submitted to Hyperledger Fabric networks need to be endorsed
+by peers that are joined to a channel before the transaction can be added to the
+ledger. Fabric peers endorse transactions by executing a smart contract using the
+inputs of the transaction proposal. The peers then sign the input and output
+generated by the smart contract execution. The endorsement policy specifies the
+set of organizations whose peers need to endorse a transaction before it can be
+added to the ledger.
+
+Each chaincode that is deployed to a channel has an endorsement policy that governs
+the assets managed by the chaincode smart contracts. However, you can override the
+chaincode level endorsement policy to create an endorsement policy for a specific key,
+either on the public channel ledger or in a private collection.
+State-based endorsement policies, also known as key-level endorsement policies, allow
+channel members to use different endorsement policies for assets that are managed by
+the same smart contract. For more information about endorsement policies and
+state-based endorsement, visit the
+[Endorsement Policies](https://hyperledger-fabric.readthedocs.io/en/release-2.2/endorsement-policies.html)
+topic in the Fabric documentation.
```

Comment:
These changes will probably conflict with the other SBE PR I just merged.

Reply:
I rebased without problem